app/assets/stylesheets/admin/components/states_decorator.scss

```diff
@@ -1,7 +1,18 @@
 @import "../variables";
 
-.state {
-  @extend .state;
+#subscriptions .state {
```

Comment:
if this is specific to subscriptions than the file name should include subscriptions.
I dont understand what .state was being extended but I think now, without the extend, the _decorator part of the filename is also gone.
So, this file should be renamed to subscritpions_state.scss or something like that ;-)